docs/sdk_performance.md

```diff
@@ -0,0 +1,57 @@
+# SDK Performance
+
+## SDK Metrics
+
+| Size   | RAM overload at peak | Batch average size | Batch max frequency |
+|--------|----------------------|--------------------|---------------------|
+| 612 KB | 6%                   | 10000 KB           | 1 second            |
+
+### Notes
+
+1. The SDK size was measured by computing the difference between the sizes of our sample application with and without
+the Datadog SDK dependency. Please have in mind that this number is volatile and it takes into 
+account also the SDK transitive dependencies. It can go up or down if your app has or not transitive
+dependencies in common with the SDK. For now the SDK uses the following transitive dependencies: 
+    -   androidx.core:core
+    -   androidx.navigation:navigation-fragment-ktx
+    -   androidx.navigation:navigation-ui-ktx
+    -   androidx.navigation:navigation-runtime-ktx
+    -   androidx.recyclerview:recyclerview
+    -   androidx.work:work-runtime
+    -   com.google.code.gson:gson
+    -   com.lyft.kronos:kronos-android
+    -   com.squareup.okhttp3:okhttp
+    -   org.jetbrains.kotlin:kotlin-stdlib
+     
+2. For the RAM overload value we profiled a sample application with/without the Datadog SDK features fully
+enabled and we measured the difference at the peak value on the RAM consumption graph.
+3. The batch average size was measured as the mean of all the batches sent in an interval of 1 minute while 
+interacting with the sample application.
+4. Once a batch is ready it will be automatically sent to Datadog's endpoint. The frequency
+at which the batches are sent starts at 20 seconds and goes up linearly with every batch sent. The max
+frequency is 1 batch/second. If there's no batch available the frequency will be linearly decreased to the
+minimum default value (20 seconds).
+
+## FAQ
+
+1. Q: **What happens when my application goes in background ?**   
+A: When the application goes in background the SDK automatically stops recording any event,
```

Comment:
What ? No!
We might stop tracking views and actions, but we can still track network requests, and we still can track spans and logs…

Reply:
I was referring more to the RUM events, I should be more specific here